metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/bolt/ThreatIntelJoinBolt.java

```diff
@@ -63,12 +65,15 @@ public ThreatIntelJoinBolt(String zookeeperUrl) {
   @Override
   public void prepare(Map map, TopologyContext topologyContext) {
     super.prepare(map, topologyContext);
+    GeoLiteDatabase.INSTANCE.update((String)getConfigurations().getGlobalConfig().get(GeoLiteDatabase.GEO_HDFS_FILE));
```

Comment:
I completely agree with this, particularly because this shows up a couple times, but without a particularly good way to avoid it that I saw.

I'm definitely open to suggestions if there's anything we want to do short term to make this a little better or clearer.